WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -315,25 +315,6 @@ - (void)showMeTab
     [self setSelectedIndex:WPTabMe];
 }
 
-- (void)showReaderTabForPost:(NSNumber *)postId onBlog:(NSNumber *)blogId
-{
-    [self showReaderTab];
-    UIViewController *topDetailVC = (UIViewController *)self.readerNavigationController.topViewController;
-
-    // TODO: needed?
```

Comment:
I think the answer is – no. It's highly unlikely it can happen. If it does, it'll just re-create the screen, so no damage done.